src/coldcard.test.js

```diff
@@ -0,0 +1,149 @@
+import {ColdcardMultisigWalletConfig, WALLET_CONFIG_VERSION} from './coldcard';
+
+describe("ColdcardMultisigWalletConfig", () => {
+  const jsonConfigUUID = {
+    name: "Test",
+    addressType: "P2SH",
+    network: "testnet",
+    quorum: {
+      requiredSigners: 2,
+      totalSigners: 3
+    },
+    startingAddressIndex: 5,
+    extendedPublicKeys: [
+      {
+        name: "unchained",
+        xpub: "tpubDF17mBZYUi35r7UEkGr7SjSisec8QL2J1zGh9WQtmnhJtHeMFy2eH2Xsnr2ynxENbqHmcEA4NnoT8T6RZxks4G5evZdWy1RbSPTm8LtNPU3",
+        bip32Path: "Unknown",
+        xfp: "83471e78"
+      },
+      {
+        name: "Os_words1",
+        xpub: "tpubDF61GHbPYRhEEsqNTDF2jbMBkEMmoksy1h2URbhZ8p7JfR9QRgbn6vkeA7g3t4Ue6uSHhYJxD9mRVz1ZQVYyW3RAPPuwVM4UeZyZPKu89DY",
+        bip32Path: "m/45'/1'/50'/0",
+        xfp: "f57ec65d"
+      },
+      {
+        name: "Os_words2",
+        xpub: "tpubDEZxsxeamoPim6T1hg2igm4vDmdQrQnHVH5gM5NjcjowtgYVv5ZhhR5sFgRVNjRFGk1HmxsFsYhu3jGaAGVCpCsL5AbAVk6xKssr6gK3tPk",
+        bip32Path: "m/45'/1'/60'/0",
+        xfp: "f57ec65d"
+      }
+    ],
+     uuid: "OWPyFOA1"
+  };
+  const coldcardConfigUUID = `# Coldcard Multisig setup file (exported from unchained-wallets)
+# https://github.com/unchained-capital/unchained-wallets
+# v${WALLET_CONFIG_VERSION}
+# 
+Name: OWPyFOA1
+Policy: 2 of 3
+Format: P2SH
+
+83471e78: tpubDF17mBZYUi35r7UEkGr7SjSisec8QL2J1zGh9WQtmnhJtHeMFy2eH2Xsnr2ynxENbqHmcEA4NnoT8T6RZxks4G5evZdWy1RbSPTm8LtNPU3\r
+f57ec65d: tpubDF61GHbPYRhEEsqNTDF2jbMBkEMmoksy1h2URbhZ8p7JfR9QRgbn6vkeA7g3t4Ue6uSHhYJxD9mRVz1ZQVYyW3RAPPuwVM4UeZyZPKu89DY\r
+f57ec65d: tpubDEZxsxeamoPim6T1hg2igm4vDmdQrQnHVH5gM5NjcjowtgYVv5ZhhR5sFgRVNjRFGk1HmxsFsYhu3jGaAGVCpCsL5AbAVk6xKssr6gK3tPk\r
+`;
+
+  const coldcardConfigName = `# Coldcard Multisig setup file (exported from unchained-wallets)
+# https://github.com/unchained-capital/unchained-wallets
+# v${WALLET_CONFIG_VERSION}
+# 
+Name: Test
+Policy: 2 of 3
+Format: P2SH
+
+83471e78: tpubDF17mBZYUi35r7UEkGr7SjSisec8QL2J1zGh9WQtmnhJtHeMFy2eH2Xsnr2ynxENbqHmcEA4NnoT8T6RZxks4G5evZdWy1RbSPTm8LtNPU3\r
+f57ec65d: tpubDF61GHbPYRhEEsqNTDF2jbMBkEMmoksy1h2URbhZ8p7JfR9QRgbn6vkeA7g3t4Ue6uSHhYJxD9mRVz1ZQVYyW3RAPPuwVM4UeZyZPKu89DY\r
+f57ec65d: tpubDEZxsxeamoPim6T1hg2igm4vDmdQrQnHVH5gM5NjcjowtgYVv5ZhhR5sFgRVNjRFGk1HmxsFsYhu3jGaAGVCpCsL5AbAVk6xKssr6gK3tPk\r
+`;
+
+  let jsonConfigCopy = JSON.parse(JSON.stringify(jsonConfigUUID));
```

Comment:
This can just be initialized empty. `let jsonConfigCopy;` looks a little cleaner and is pretty standard in my experience. It'll get assigned in the beforeEach.

Reply:
Got it